api/views.py

```diff
@@ -41,3 +49,28 @@ def get(self, request, format=None):
                                   .prefetch_related('person_set')
         serializer = ExportInstructorLocationsSerializer(airports, many=True)
         return Response(serializer.data)
+
+
+class ListEvents(APIView):
+    # I wanted to use ListAPIView, but it had problems with the way we test
+    # this code...
+    serializer_class = EventSerializer
+    permission_classes = (IsAuthenticatedOrReadOnly, )
+    queryset = None  # override this in the subclass
+
+    def get(self, request, format=None):
+        objects = self.queryset.all()
+        serializer = self.serializer_class(objects, many=True)
+        return Response(serializer.data)
+
+
+class PastEvents(ListEvents):
+    queryset = Event.objects.past_events().order_by('-start')
```

Comment:
That's the code we're using here (`.past_events()`, `.ongoing_events()`, `.upcoming_events()`).